skins/oasis/css/core/ads.scss

```diff
@@ -284,7 +284,7 @@ body > a > img[src$="noad.gif"] {
 		position: absolute;
 	}
 
-	#INCONTENT_BOXAD_1, #incontent_boxad_1 {
+	.gpt-ad {
```

Comment:
makes sense to go with something less specific in this place as we're wrapping it above with specific `#WikiaAdInContentPlaceHolder` 👍